src/catchup/CatchupWork.cpp

```diff
@@ -53,10 +53,27 @@ CatchupWork::~CatchupWork()
 std::string
 CatchupWork::getStatus() const
 {
-    if (mCatchupSeq)
+    if (mApplyBufferedLedgersWork)
```

Comment:
I synced with @marta-lokhova about this, and we're in agreement.

the status string generated should really be something like "Catching up to ledger {}: {}", with
* first param derived from `mCatchupConfiguration`
* second param derived from `mStatusWork->getStatus()` (or "starting"), where `mStatusWork` is set in `doReset` (to null) and `runCatchupStep` to "whatever work we want to track right now"
    * this avoids having to keep this function `getStatus` in sync with whatever is happening in `runCatchupStep`
    * "what we track" is easy to derive in the other function, it's usually the last work created (or in the case of high level sequence, track the sequence) or when we return `foo->getState()` it means we want to track `foo`

note that we should remove the "Catching up" string from `CatchupManager` so that we avoid duplicating status.

Reply:
Done